mirrord-layer/src/file/hooks.rs

```diff
@@ -136,7 +136,7 @@ pub(super) unsafe extern "C" fn fdopen_detour(fd: RawFd, raw_mode: *const c_char
 /// `open_detour`.
 /// `fd` for a file descriptor with the `O_DIRECTORY` flag.
 #[hook_guard_fn]
-#[tracing::instrument(level = "trace", skip(raw_path))]
+#[tracing::instrument(level = "debug", skip(raw_path))]
```

Comment:
prefer to remove this all together (tracing directly from hook....), same for above if already touched.

Reply:
Removed!